docs/features/correlation.md

```diff
@@ -18,4 +18,124 @@ This feature requires to install our NuGet package
 PM > Install-Package Arcus.Observability.Correlation
 ```
 
+## What We Provide
+
+The `Arcus.Observability.Correlation` library provides a way to get access to correlation information across your application.
+What it **DOES NOT** provide is how this correlation information is initially set.
```

Comment:
We changed this, no?

Reply:
Well, the place where you set the correlation is specific to the application, no? (Web Api = middleware + HTTP features, Message pump = OnReceive)  So, I would leave it.